lite/backends/imagination_nna/imgdnn_manager.h

```diff
@@ -0,0 +1,253 @@
+// Copyright (c) 2019 PaddlePaddle Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <unistd.h>
+#include <cmath>
+#include <memory>
+#include <string>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+#include "imgdnn.h"  // NOLINT
+#include "lite/core/op_lite.h"
+#include "lite/core/tensor.h"
+
+namespace paddle {
+namespace lite {
+namespace imagination_nna {
+
+class ImgdnnManager {
+  imgdnn_err_code err_;
+  imgdnn_device device_;
+  imgdnn_network net_{nullptr};
+  imgdnn_context context_{nullptr};
+  imgdnn_binding binding_{nullptr};
+  imgdnn_network_object net_obj_{nullptr};
+  std::vector<uint8_t *> coef_pool;
+
+  imgdnn_tensor convertQuantTensorType(imgdnn_tensor a_tensor,
```

Comment:
Please follow google c++ code style, https://google.github.io/styleguide/cppguide.html#Function_Names

except the APIs from the third-parties.


Applicable to all the other functions in this PR.

Reply:
Done